src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -167,14 +167,18 @@ namespace chip {
 #endif
 #endif // CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST
 
+constexpr System::Clock::Milliseconds32 kDefaultActiveTime = System::Clock::Milliseconds16(4000);
+
 /**
  *  @brief
  *    The ReliableMessageProtocol configuration.
  */
 struct ReliableMessageProtocolConfig
 {
-    ReliableMessageProtocolConfig(System::Clock::Milliseconds32 idleInterval, System::Clock::Milliseconds32 activeInterval) :
-        mIdleRetransTimeout(idleInterval), mActiveRetransTimeout(activeInterval)
+    ReliableMessageProtocolConfig(System::Clock::Milliseconds32 idleInterval, System::Clock::Milliseconds32 activeInterval,
+                                  System::Clock::Milliseconds16 activeThreshold = kDefaultActiveTime) :
```

Comment:
is setting default value useful since we are overriding it the next line?

Reply:
This allows users to not pass a value for the activeThreshold. For example, users that used the previous form and do not use ICD. They can keep the constructor they had. It will set activeThreshold to its default in this case. Anyone calling